docs/query-builder.txt

```diff
@@ -9,588 +9,1095 @@ Query Builder
+        },
+        {
+           "_id": { "runtime": 67 },
+           "runtime": 67,
+           "title": "Bartok the Magnificent"
+         },
+         {
+           "_id": { "runtime":68 },
+           "runtime": 68,
+           "title": "Mickey's Twice Upon a Christmas"
+         },
+         ...
+      ]
+
+.. _laravel-query-builder-aggregation-count:
+
+Number of Results Example
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following example shows how to use the ``count()``
+query builder method to return the number of documents
+contained in the ``movies`` collection:
+
+.. literalinclude:: /includes/query-builder/MovieController.php
+   :language: php
+   :dedent:
+   :start-after: begin aggregation count
+   :end-before: end aggregation count
+
+.. _laravel-query-builder-aggregation-max:
+
+Maximum Value of a Field Example
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following example shows how to use the ``max()``
+query builder method to return the highest numerical
+value of the ``runtime`` field from the entire
+``movies`` collection:
+
+.. literalinclude:: /includes/query-builder/MovieController.php
+   :language: php
+   :dedent:
+   :start-after: begin aggregation max
+   :end-before: end aggregation max
+
+.. _laravel-query-builder-aggregation-min:
+
+Minimum Value of a Field Example
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following example shows how to use the ``min()``
+query builder method to return the lowest numerical
+value of the ``year`` field from the entire ``movies``
+collection:
+
+.. literalinclude:: /includes/query-builder/MovieController.php
+   :language: php
+   :dedent:
+   :start-after: begin aggregation min
+   :end-before: end aggregation min
+
+.. _laravel-query-builder-aggregation-avg:
+
+Average Value of a Field Example
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following example shows how to use the ``avg()``
+query builder method to return the numerical average, or
+arithmetic mean, of the ``imdb.rating`` values from
+the entire ``movies`` collection.
+
+.. literalinclude:: /includes/query-builder/MovieController.php
+   :language: php
+   :dedent:
+   :start-after: begin aggregation avg
+   :end-before: end aggregation avg
+
+.. _laravel-query-builder-aggregation-sum:
+
+Summed Value of a Field Example
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following example shows how to use the ``sum()``
+query builder method to return the numerical total of
+the ``imdb.votes`` values from the entire ``movies``
+collection:
+
+
+.. literalinclude:: /includes/query-builder/MovieController.php
+   :language: php
+   :dedent:
+   :start-after: begin aggregation sum
+   :end-before: end aggregation sum
+
+.. _laravel-query-builder-aggregate-matched:
+
+Aggregate Matched Results Example
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following example shows how to use the ``sum()``
+query builder method to return the numerical total of
+the ``imdb.votes`` values from movies released in
+after the year "2000":
```

Comment:
I: description does not match the example

Reply:
Good catch, thanks!